modules/unsupported/gml-streaming/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0                                 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.geotools</groupId>
+    <artifactId>unsupported</artifactId>
+    <version>26-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>gt-gml-streaming</artifactId>
+  <packaging>jar</packaging>
+  <name>GML XML support using Streaming API for XML (StAX)</name>
+  <description>Efficiently read GML elements using StAX in a streaming manner. No writing.</description>
```

Comment:
At first I though the whole feature reading was moved, but I see only gml geometries parsing is implemneted?
GML is a huge spec, covers even more than features, but it's commonly associated to features.

The module description should be more specific and clarify only geometry parsing is supported. If there is no intention to parse features and feature collections before graduating it, the name of the module should also be changed, as the focus is very specific, unlike the more complex parsers available in GeoTools.

I guess it's also version specific, as geometry encoding is the part that changed over time. 
A clearer name might be ``gt-gml3-geometry-streaming``, if that's the ambition of the module.

Reply:
Good points, I will clarify it's only parsing geometries. I do not have the intention to add feature and featureCollection parsing to it and definitely not general application schemas. This would keep this module small and simple but still useful. I'll see if I can write some user guide documentation to clarify the intention and when this parser can be used.

Not sure if the '3' suffix is needed in `gt-gml3-geometry-streaming`, because for the elements it supports it will parse GML 2.x elements which are deprecated in GML 3.x but retained for backwards compatibility. A clearer distinction would be GML 3.2 which has a different namespace. I'll have to look into the specific geometry element differences, but I think this parser could support both the `http://www.opengis.net/gml` and `http://www.opengis.net/gml/3.2` namespaces.

Reply:
Good point indeed, no version number needed if it can succesfully parse geometries from all GML versions.